scripts/apps/desks/directives/DesksReactDropdown.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.desks
+ * @name DesksReactDropdown
+ *
+ * @requires React
+ * @requires item
+ * @requires className
+ * @requires desks
+ * @requires noDesksLabel
+ *
+ * @param {Object} [desks] collection of desks
+ *
+ * @description Creates dropdown react element with list of available desks
+ */
+
+import React from 'react';
+
+DesksReactDropdown.$inject = ['item', 'className', 'desks', 'gettext', 'noDesksLabel'];
```

Comment:
Can we turn this whole file into an actual React component implemented as an ES6 class that extends `React.Component`. We can document it using `@ngdoc React`. Also, you can safely use JSX, it works fine. No need to programatically create the elements.

Also, please use `propTypes` to document component properties.